src/main/kotlin/no/nav/syfo/behandlerdialog/api/BehandlerdialogApi.kt

```diff
@@ -0,0 +1,43 @@
+package no.nav.syfo.behandlerdialog.api
+
+import io.ktor.http.*
+import io.ktor.server.application.*
+import io.ktor.server.request.*
+import io.ktor.server.response.*
+import io.ktor.server.routing.*
+import no.nav.syfo.application.api.VeilederTilgangskontrollPlugin
+import no.nav.syfo.behandlerdialog.api.domain.BehandlerdialogDTO
+import no.nav.syfo.client.veiledertilgang.VeilederTilgangskontrollClient
+import no.nav.syfo.domain.PersonIdent
+import no.nav.syfo.util.*
+
+const val behandlerdialogApiBasePath = "/api/internad/v1/behandlerdialog"
+const val behandlerdialogApiPersonidentPath = "/personident" // TODO: Noe annet enn /personident eller?
+
+private const val API_ACTION = "access behandlerdialog for person"
+
+fun Route.registerBehandlerdialogApi(
+    veilederTilgangskontrollClient: VeilederTilgangskontrollClient,
+) {
+    route(behandlerdialogApiBasePath) {
+        install(VeilederTilgangskontrollPlugin) {
+            this.action = API_ACTION
+            this.veilederTilgangskontrollClient = veilederTilgangskontrollClient
+        }
+
+        get(behandlerdialogApiPersonidentPath) {
+            // Hent ut fra databasen her
+            call.respond(HttpStatusCode.OK)
+        }
+
+        post(behandlerdialogApiPersonidentPath) {
+            val personIdent = call.personIdent()
+            val requestDTO = call.receive<BehandlerdialogDTO>()
+            // Lagre i db, legg til i joark-cronjob og send på kafka her
+            call.respond(requestDTO)
+        }
```

Comment:
Bare sender dette rett gjennom i første iterasjon.

Reply:
Funker like greit med bare call.respond(HttpStatusCode.OK) kanskje? Så kan man la den være, men legge til lagring osv senere. Eller tenker du vi vil returnere request-dtoen tilbake?

Reply:
Nei, tenkte at den skal returnere OK! Dette var mest for å sjekke at ting "gikk gjennom" i testene så jeg kunne verifisere feltene fra DTOen i testen.